.chloggen/gcp-0-35-2.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: googlecloudexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Bug fixes for GCP exporters.
+
+# One or more tracking issues related to the change
+issues: [18755]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: Fixes infinite loop/memory leak in GCP logging exporter. Also updates credentials config options.
```

Comment:
Why not put this directly on the `note`? If you read the changelog you will know that this is a bugfix already because of the `change_type`.